CalibParam:  
  # 内参校准参数（已更新）
  INTRI:
    Camera:
      - key: /image_topic
        value:
          polymorphic_id: 2147483649
          polymorphic_name: pinhole_brown_t2
          ptr_wrapper:
            id: 2147483650
            data:
              img_width: 1280
              img_height: 1024
              # focal_length:                     # 8mm相机
              #   - 1740.1391026563904
              #   - 1743.466162735272
              # principal_point:
              #   - 600.0573645949644
              #   - 415.5957223704912
              # disto_param:
              #   - -0.11407724406859104
              #   - 0.9907393620859306
              #   - -0.012350441691573684
              #   - -0.0022510643311303606
              #   - -2.920015848738005
              # focal_length:                     # 6mm相机
              #   - 1330.649408921195
              #   - 1332.3006999890033
              # principal_point:
              #   - 627.2791611048931
              #   - 533.2806702069377
              # disto_param:
              #   - -0.14828277112729327
              #   - 0.7868245788660153
              #   - 0.0023478362007317123
              #   - 0.002535817596959765
              #   - -1.5222934363748177
              focal_length:                   # 4mm相机
                - 893.4306170676092
                - 888.878979439582
              principal_point:
                - 618.4622245880115
                - 511.73226091888046
              disto_param:
                - -0.09613598816721357
                - 0.3686128765511766
                - -0.0035685183642176046
                - 0.002876545526120485
                - -0.594006385480322
  

# 相机参数
camera:
  type: "hik"  # 可选: "hik", "mindvision"
  parameters:
    exposure_ms: 2.0
    gain: 12.0
    gamma: 0.6
    fps: 120.0
    target_width: 1280
    target_height: 1024
    image_topic: "image_topic"
    vid_pid: "2bdf:0283"  # MindVision相机的VID:PID
    debug: false


# lower_Dart通讯参数
lower_Dart:
enable_serial: true
# com_port: "/dev/ttyUSB0"
com_port: "/dev/ttyACM0"


# video视频测试参数
Video:
video_path: "/home/guo/ITL_Auto_aim/test/buff_simulation_record.mp4"

  
# 基地Base识别参数
Base_Hit:
  Openvino_XML: /home/guo/ITL_Auto_aim/src/model/Katrin.xml
  Openvino_Deveice: "CPU"
  input_width: 640
  input_height: 384
  score_threshold: 0.5
  nms_threshold: 0.1

# LightTracker 参数
LightTracker:
  min_detect_count: 3           # 最小检测次数（detecting → tracking）
  max_temp_lost_count: 10       # 最大临时丢失次数（temp_lost → lost）


LightAimer:
  begin_x: 618.46               # 基准点x坐标
  base_offset: 0              # 基础补偿
  offsets:                    # 根据number的补偿表
    1: 5                      # 飞镖1的补偿
    2: -6                     # 飞镖2的补偿
    3: -4                     # 飞镖3的补偿
    4: -6                     # 飞镖4的补偿